src/main/java/racing_car/step03/Validation.java

```diff
@@ -0,0 +1,13 @@
+package racing_car.step03;
+
+public class Validation {
+    public static void inputValidation(int carCount , int tryCount){
+        if (carCount<=0){
```

Comment:
```suggestion
        if (carCount<=0) {
```
전체적으로 컨벤션 관련해서 검토부탁드려요 :)
option + cmd + L 을 이용하여 자동으로 정리할 수도 있으니 참고하시면 좋을거같아요!